README.md

```diff
@@ -16,6 +16,12 @@ Path to inspections file relative to root of project directory.
 
 **Required**
 
+### `ignore_file_pattern`
+
+A string of space-separated regular expressions of files whose inspection results are to be ignored.
```

Comment:
@kumar-shrivatsav If you are going to support multiple patterns, I would suggest not using space separated. you may well want spaces in your regex.

Ask people to make it a JSON array (or some such proper serialization format), with each element as a regex.

Reply:
@gps 
We can take the input in json format as string: 
```
{ 
     1:".*src/generated.*",
     2: ".*basedb/src.*"
}
``` 
Here, the key is not useful. Only the value holds the regex pattern values. 

Another way is to read a python list as string: `'[" .*src/generated.*",".*basedb/src.* "]'`
We can then convert it back to list type in code. 
which approach do you suggest?

Reply:
@kumar-shrivatsav would something like this work?

```yml
with:
  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  INSPECTIONS_FILE: Inspections.xml
  IGNORE_FILE_PATTERN: |
    [
      "foobar",
      "blah",
      "baz"
    ]
```